x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/components/package_card.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiCard, EuiIcon, ICON_TYPES } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { PackageInfo, PackageListItem } from '../../../../../../common/types/epm';
```

Comment:
Instead of deeply traversing into `/common`, you can import `/common` into [`/public`](https://github.com/elastic/kibana/blob/feature-ingest/x-pack/plugins/ingest_manager/public/applications/ingest_manager/types/index.ts), so this line becomes:
`import { PackageInfo, PackageListItem } from '../../../types';`

This comment applies to all the imports from `/common` within the rest of this PR's UI code 🙂